Makefile

```diff
@@ -1,7 +1,5 @@
 test:
-	nvim --headless --clean -u tests/minimal_init.vim -c "TSUpdateSync org" -c "PlenaryBustedDirectory tests/plenary/ {minimal_init = 'tests/minimal_init.vim'}"
-testfile:
-	nvim --headless --clean -u tests/minimal_init.vim -c "TSUpdateSync org" -c "PlenaryBustedDirectory $(FILE) {minimal_init = 'tests/minimal_init.vim'}"
+	nvim --headless --clean -u tests/test.lua "$(FILE)"
```

Comment:
While we are at this, could we introduce a `make clean` command that would clean up the `.deps` and all other settings/deps for the tests? I had this issue before where my local tests were failing because I had outdated version of `nvim-treesitter` locally so I had to go inside it and do a `git pull`. If we introduce a `make clean`, we can run that and just re-run `make test` to start fresh.

Reply:
Sure, no problem.

Don't worry about "going back and forth", all part of the process. It would be worse if this stuff didn't get ironed out ahead of time. 😉

Reply:
Done, take a look. Instead of doing a git pull, I just nuke the directory. The test command will redownload it at startup if it doesn't exist.